zerver/tests/test_digest.py

```diff
@@ -159,6 +159,14 @@ def test_guest_user_multiple_stream_sender(
         new_stream_names = kwargs["context"]["new_streams"]["plain"]
         self.assertTrue("web_public_stream" in new_stream_names)
 
+    def test_no_logging(self) -> None:
+        hamlet = self.example_user("hamlet")
+        RealmAuditLog.objects.all().delete()
+        bulk_write_realm_audit_logs([])
+        self.assert_length(RealmAuditLog.objects.all(), 0)
+        bulk_write_realm_audit_logs([hamlet])
+        self.assert_length(RealmAuditLog.objects.all(), 1)
```

Comment:
Do we need to pre-delete, or could we just check that the total number goes up by one, or check that a RealmAuditLog of the right type is created? Generally we try to avoid tests that delete all of an object from the database, since it can be slow if there's a lot of them.

Reply:
DONE - good call. is it obvious I blindly copied+modified?  sorry, I'll try to do better next time!